hw/top_earlgrey/data/ip/chip_otbn_testplan.hjson

```diff
@@ -0,0 +1,147 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: otbn
+  testpoints: [
+    {
+      name: chip_sw_otbn_isatest
+      descr: '''Run every OTBN instruction and check final state.
+
+                Run the existing OTBN smoke test which uses every instruction
+                (hw/ip/otbn/dv/smoke/smoke_test.s). Check the final register state matches the
+                expected values (in hw/ip/otbn/dv/smoke/smoke_expected.txt). Check OTBN completed
+                successfully without errors.
+
+                Following the smoke test run a new program just to dump out the register state to
+                dmem, check the values do not match the previous values from the smoke test run to
+                ensure the internal state secure wipe has worked.
+             '''
+      stage: V2
+      si_stage: SV1
+      tests: [""]
```

Comment:
This is a list with one item, which happens to be the empty string. I think you want `[]`.